code/API_definitions/carrier_billing.yaml

```
@@ -991,91 +835,147 @@ components:
         productId:
           type: string
           description: The product identifier to be combined with the `serviceId` to uniquely identify the product being purchased. For example if the `serviceId` relates to a VOD service, the `productId` can specify the movie rented
-    ErrorInfo:
-      type: object
+    CloudEvent:
+      description: The notification format
       required:
-        - code
-        - message
-        - status
+        - id
+        - source
+        - specversion
+        - type
+        - time
       properties:
-        code:
+        id:
           type: string
-          description: Code given to this error
-        status:
+          description: Identifier of this event, that must be unique in the source context.
+          example: sd5e-uy52-88t4-za66
+        source:
+          $ref: "#/components/schemas/Source"
+        type:
           type: string
-          description: HTTP response status code
-        message:
+          description: Type of event as defined in each CAMARA API
+          example: org.camaraproject.carrier-billing.v0.payment-reserved
+        specversion:
           type: string
-          description: Detailed error description
-    EventNotification:
-      type: object
-      description: Generic CAMARA data type for event notification structure
-      required:
-        - event
-      properties:
-        eventSubscriptionId:
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          example: 1.0
+        datacontenttype:
           type: string
-          description: subscription identifier
-        event:
-          $ref: "#/components/schemas/Event"
-    Event:
+          description: 'media-type that describes the event payload encoding, must be "application/json" for CAMARA APIs'
+          example: application/json
+        data:
+          type: object
+          description: Event details payload described in each CAMARA API and referenced by its type
+        time:
+          $ref: "#/components/schemas/DateTime"
+      discriminator:
+        propertyName: "type"
+        mapping:
+          org.camaraproject.carrier-billing.v0.payment-reserved: "#/components/schemas/EventPaymentReserved"
+          org.camaraproject.carrier-billing.v0.payment-completed: "#/components/schemas/EventPaymentCompleted"
+          org.camaraproject.carrier-billing.v0.payment-cancelled: "#/components/schemas/EventPaymentCancelled"
+    Source:
+      type: string
+      format: uri-reference
+      minLength: 1
+      description: Identifies the context in which an event happened in the specific Provider Implementation.
+      example:
+        - https://github.com/cloudevents
+        - mailto:cncf-wg-serverless@lists.cncf.io
+        - urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66
+        - cloudevents/spec/pull/123
+        - /sensors/tn-1234567/alerts
+        - 1-555-123-4567
```

Here we should give one example or multiple examples  - it should be updated in Commonalities

It can be:

-  Absolute URI like: https://notificationSendServer12.supertelco.com starting with **uri-scheme**
-  Relative Reference like: /supertelco/notification/SendServer12
```suggestion
example: https://notificationSendServer12.supertelco.com)
```

---

Fine to have one example. Will also add "" as being a string